### PR TITLE
Compatibility with Laravel 11

### PR DESCRIPTION
**Pull request by sharafat**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,15 +25,15 @@
     ],
     "require": {
         "php": "^7.2|^8.0.2|^8.1",
-        "laravel/framework": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*|7.*|8.*|^9.0|^10.0",
-        "eklundkristoffer/seedster": "^7.0",
+        "laravel/framework": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*|7.*|8.*|^9.0|^10.0|^11.0",
+        "eklundkristoffer/seedster": "^8.0",
         "laravel/helpers": "^1.5"
```
Comment (angusm73) [inline composer.json:30]: Might need to bump `laravel/helpers` up to [v1.7](https://packagist.org/packages/laravel/helpers#v1.7.0)

File: composer.json [hunk 2]
```diff
@@ -25,15 +25,15 @@
...
     },
     "require-dev": {
         "orchestra/testbench": "^6.0|^7.0|^8.0",
         "laravel/tinker": "^2.7",
```
Comment (angusm73) [inline composer.json:34]: Might also want to bump this dev package `laravel/tinker` up to [v2.9](https://packagist.org/packages/laravel/tinker#v2.9.0)

File: composer.json [hunk 3]
```diff
@@ -25,15 +25,15 @@
...
     "require": {
         "php": "^7.2|^8.0.2|^8.1",
-        "laravel/framework": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*|7.*|8.*|^9.0|^10.0",
-        "eklundkristoffer/seedster": "^7.0",
+        "laravel/framework": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*|7.*|8.*|^9.0|^10.0|^11.0",
+        "eklundkristoffer/seedster": "^8.0",
         "laravel/helpers": "^1.5"
     },
     "require-dev": {
         "orchestra/testbench": "^6.0|^7.0|^8.0",
```
Comment (angusm73) [inline composer.json:33]: Also consider adding `|^9.0` to `orchestra/testbench` versions

File: composer.json [hunk 4]
```diff
@@ -25,15 +25,15 @@
     ],
     "require": {
         "php": "^7.2|^8.0.2|^8.1",
```
Comment (angusm73) [inline composer.json:27]: Laravel 11 also requires PHP 8.2 😃
Consider appending `|^8.2` to these versions, or replace both version 8's with `^8` to support any PHP 8 installation.

**Review by angusm73 — APPROVED** [review]
Looks good! Just a couple suggestions for other packages to bump 🚀

File: composer.json (hunk 1 above)
Comment (angusm73) [reply composer.json:30]: ^ for Laravel 11 support

**Comment by sharafat** [comment]
@angusm73 thanks so much for your careful review and helpful suggestions! 😄 Although for most of the cases the `^` took care of the minor version compatibility and hence this package works in my application without any issues with the minimal changes I had made, I do find your suggestions useful to implement. So here's the updated version!

**Review by jeremykenedy — APPROVED** [review]

**Comment by sharafat** [comment]
@angusm73 thanks for merging! Could you please also tag a release? Thanks! 🚀

**Pull request merged**